stream.go

```diff
@@ -66,45 +72,182 @@ func (f *File) NewStreamWriter(sheet string) (*StreamWriter, error) {
 		File:    f,
 		Sheet:   sheet,
 		SheetID: sheetID,
+		replaceMap: map[string][]byte{
+			"XMLName": []byte{},
+		},
 	}
+	rsw.encoder = xml.NewEncoder(&rsw.SheetData)
 	rsw.SheetData.WriteString("<sheetData>")
 	return rsw, nil
 }
 
-// SetRow writes an array to streaming row by given worksheet name, starting
-// coordinate and a pointer to array type 'slice'. Note that, cell settings
-// with styles are not supported currently and after set rows, you must call the
-// 'Flush' method to end the streaming writing process. The following
-// shows the supported data types:
+// SetTable creates an Excel table for the StreamWriter. This function must be
```

Comment:
Please add the using example in the doc.

Reply:
I have completely re-worked tables. I added an example in the new `AddTable` function.